Tests/Packet++Test/Tests/SomeIpTests.cpp

```diff
@@ -0,0 +1,115 @@
+#include "../TestDefinition.h"
+#include "../Utils/TestUtils.h"
+#include "EthLayer.h"
+#include "IPv6Layer.h"
+#include "Packet.h"
+#include "SomeIpLayer.h"
+#include "SystemUtils.h"
+#include "UdpLayer.h"
+#include "VlanLayer.h"
+#include <array>
+#include <cstring>
+
+PTF_TEST_CASE(SomeIpParsingTest)
```

Comment:
We should also test the following class methods:
- `getPduPayload()`
- `getPduPayloadSize()`
- `toString()`
- `getHeaderLen()`